fpo-web/src/app/survey/survey.component.ts

```diff
@@ -271,8 +271,22 @@ export class SurveyComponent implements OnInit, OnDestroy {
   }
 
   submitForm(data: any) {
-    //Default to notice to disputant response. Please change later on. 
-    const url = this.dataService.getApiUrl("submit-form/?name=notice-to-disputant-response");
+    // This is horrendous.
+    if (
+      (data.continueDispute == 'y' &&
+      data.disputeType == 'fineAmount' &&
+      data.disputeInWriting == 'y')
+        ||
+      (data.continueDispute == 'n' &&
+      data.moreTimeToPay1 == 'y'
+      )
+    ) {
+      var form = "violation-ticket-statement-and-written-reasons"; 
```

Comment:
Use `let form` before the conditional block and assign `form` inside the branches. `var` is not generally used in typescript code.